manifests/alert_cycle.pp

```diff
@@ -0,0 +1,40 @@
+# @summary Pauses and starts (cycles) alerts on a schedule
+#
+# @param crons
+#   Hash of CRON entries for pausing/starting alerts.
+#   For the command key, the path to a alert_toggle script should be omitted
+#
+# @param cycle_alerts
+#   Enable or disable the cycling of alerts
+#
+# @param script_path
+#   Path to the alert_toggle.sh script
+#   
+class profile_metrics_alerting::alert_cycle (
+  Hash $crons,
+  Boolean $cycle_alerts,
```

Comment:
I'd prefer to name this variable something like `enable_cycle_alerts`.

It was not obvious to me reading about this parameter that this was a boolean that toggles whether or not the cycling of alerts was enabled.

Reply:
Good point, I've renamed it to `enable_cycle_alerts`